ksql-version-metrics-client/src/main/java/io/confluent/ksql/version/metrics/KsqlVersionCheckerAgent.java

```diff
@@ -19,48 +19,67 @@
 import io.confluent.ksql.version.metrics.collector.KsqlModuleType;
 import io.confluent.support.metrics.BaseSupportConfig;
 import io.confluent.support.metrics.PhoneHomeConfig;
+import java.time.Clock;
+import java.util.Objects;
 import java.util.Properties;
+import java.util.concurrent.TimeUnit;
+import java.util.function.Supplier;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class KsqlVersionCheckerAgent implements VersionCheckerAgent {
 
-  private KsqlVersionChecker ksqlVersionChecker;
+  private static final long MAX_INTERVAL = TimeUnit.DAYS.toMillis(1);
 
-  private boolean enableSettlingTime;
+
+  private final boolean enableSettingTime;
```

Comment:
Looks like the parameter (as defined in BaseMetricsReporter) is called `enableSettlingTime`. Can we add the `l` back in to avoid confusion?

Reply:
Oops! Good point! I'll change it back!